src/main/java/io/kestra/plugin/notifications/sentry/SentryAlert.java

```diff
@@ -0,0 +1,147 @@
+                            "Execution ID": "{{ execution.id }}",
+                            "Execution Status": "{{ execution.state.current }}",
+                            "Link": "{{link}}"
+                          }
+                      }
+                """
+        ),
+        @Example(
+            title = "Send a Sentry alert",
+            full = true,
+            code = """
+                id: sentry_alert
+                namespace: dev
+
+                tasks:
+                  - id: send_sentry_message
+                    type: io.kestra.plugin.notifications.sentry.SentryAlert
+                    dsn: "{{ secret('SENTRY_DSN') }}" # format: https://{PUBLIC_KEY}@{HOST}/{PROJECT_ID}
+                    payload: |
+                      {
+                          "event_id": "fc6d8c0c43fc4630ad850ee518f1b9d0",
+                          "timestamp": "{{ execution.startDate }}",
+                          "platform": "java",
+                          "level": "info",
+                          "transaction": "/execution/id/{{ execution.id }}",
+                          "server_name": "localhost:8080",
+                          "extra": {
+                            "Namespace": "{{ flow.namespace }}",
+                            "Flow ID": "{{ flow.id }}",
+                            "Execution ID": "{{ execution.id }}",
+                            "Execution Status": "{{ execution.state.current }}",
+                            "Link": "{{link}}"
+                          }
+                      }
+                """
+        ),
+    }
+)
+public class SentryAlert extends Task implements RunnableTask<VoidOutput> {
+
+    public static final String SENTRY_VERSION = "7";
+    public static final String SENTRY_CLIENT = "java";
+    public static final String SENTRY_DSN_REGEXP = "^(https?://[a-f0-9]+@o[0-9]+\\.ingest\\.sentry\\.io/[0-9]+)$";
+
+    @Schema(
+        title = "Sentry DSN"
+    )
+    @PluginProperty(dynamic = true)
+    @NotBlank
+    protected String dsn;
+
+    @Schema(
+        title = "Sentry event payload"
+    )
+    @PluginProperty(dynamic = true)
+    protected String payload;
+
+    @Override
+    public VoidOutput run(RunContext runContext) throws Exception {
+        String dsn = runContext.render(this.dsn);
+
+        String url = dsn;
+        if (dsn.matches(SENTRY_DSN_REGEXP)) {
+            String protocol = dsn.split("://")[0];
+            String publicKey = dsn.split("@")[0].replace(protocol + "://", "");
+            String host = dsn.split("@")[1].split("/")[0];
+            String projectId = dsn.split("@")[1].split("/")[1];
+
+            url = "%s://%s/api/%s/store/?sentry_version=%s&sentry_client=%s&sentry_key=%s" // https://{HOST}/api/{PROJECT_ID}/store/?sentry_version=7&sentry_client=java&sentry_key={PUBLIC_KEY}
+                .formatted(protocol, host, projectId, SENTRY_VERSION, SENTRY_CLIENT, publicKey);
+        }
```

Comment:
I struggle to understand what's this is about, can you explain it to me?

Reply:
I think the URL is confusing and Nikita wanted to extract info from DSN to make it easier

agree worth adding a comment to validate 👍

Reply:
Yes, i didn't find documentation where users can easily find url, project id, and public key, and form it into api request. So it's easier to implement it like that (By parsing DSN it's how they implement in own Sentry SDK)

Reply:
@iNikitaGricenko can you add a comment before the URL explaining that?

can be something like: "To make passing the correct API endpoint URL easier, users only need to provide the Sentry DSN, and we parse the required attributes for the URL"